__tests__/serverjs/cards.test.js

```diff
@@ -109,106 +110,108 @@ test("cardFromId returns a placeholder card object when given a nonexistent ID",
+    parsed_cost: ['b', 'b', '2'],
+    power: '3',
+    promo: true,
+    rarity: 'mythic',
+    scryfall_uri: 'https://scryfall.com/card/celd/356/rankle-master-of-pranks?utm_source=api',
+    set: 'celd',
+    type: 'Legendary Creature — Faerie Rogue',
   };
   var promise = carddb.initializeCardDb(fixturesPath, true);
   return promise.then(function() {
     const result = carddb.getCardDetails({
-      "cardID": _id
+      cardID: _id,
     });
     expect(result).toEqual(expected);
   });
 });
 
-test("getCardDetails returns a placeholder card object when given a nonexistent ID", () => {
+test('getCardDetails returns a placeholder card object when given a nonexistent ID', () => {
   expect.assertions(1);
-  const _id = "not real";
+  const _id = 'not real';
   var expected = placeholderCard;
   expected._id = _id;
   var promise = carddb.initializeCardDb(fixturesPath, true);
   return promise.then(function() {
     const result = carddb.getCardDetails({
-      "cardID": _id
+      cardID: _id,
     });
     expect(result).toEqual(expected);
   });
 });
 
-test("normalizedName normalized ascii correctly", () => {
-  const rawName = "Garruk, Primal Hunter";
-  const expected = "garruk, primal hunter";
+test('normalizedName normalized ascii correctly', () => {
+  const rawName = 'Garruk, Primal Hunter';
+  const expected = 'garruk, primal hunter';
   const result = carddb.normalizedName({
-    "name": rawName
+    name: rawName,
   });
   expect(result).toBe(expected);
 });
 
-test("normalizedName normalizes unicode correctly", () => {
-  const rawName = "Ætherspouts";
-  const expected = "ætherspouts";
+test('normalizedName normalizes unicode correctly', () => {
+  const rawName = 'Ætherspouts';
+  const expected = 'ætherspouts';
   const result = carddb.normalizedName({
-    "name": rawName
+    name: rawName,
   });
   expect(result).toBe(expected);
 });
 
-test("allIds correctly maps a cardname to an ID", () => {
+test('allIds correctly maps a cardname to an ID', () => {
   expect.assertions(2);
   var promise = carddb.initializeCardDb(fixturesPath, true);
   return promise.then(function() {
-    const expected = "ee4d196e-7ce4-4dc1-9d58-102a89aca2a4";
+    const expected = 'ee4d196e-7ce4-4dc1-9d58-102a89aca2a4';
     const result = carddb.allIds({
-      "name": "Rankle, Master of Pranks"
+      name: 'Rankle, Master of Pranks',
     });
     expect(result.length).toBe(1);
     expect(result[0]).toBe(expected);
   });
 });
 
-test("loadJSONFile loads a JSON file into the correct attribute", () => {
+test('loadJSONFile loads a JSON file into the correct attribute', () => {
   expect.assertions(1);
-  const attribute = "testAttribute";
-  return carddb.loadJSONFile(fixturesPath + "/names.json", attribute).then(function() {
+  const attribute = 'testAttribute';
+  return carddb.loadJSONFile(fixturesPath + '/names.json', attribute).then(function() {
```

Comment:
should this not be:

```
return carddb.loadJSONFile(`${fixturesPath}/names.json`, attribute).then(function() {
```

Reply:
Generally yes but I don't think prettier does that transformation. We've got a lot of code written in the sums-of-strings style.

Reply:
that's fair... this looks really good @phulin